priv/repo/seeds.exs

```diff
@@ -9,3 +9,12 @@
 #
 # We recommend using the bang functions (`insert!`, `update!`
 # and so on) as they will fail if something goes wrong.
+defmodule MtaScheduledArrivalTime do
+  def seed do
+    unless MindTheGapp.Mta.ScheduledArrivalTime.is_data_loaded?() do
```

Comment:
It is conventional that database seeds are only run once when the database is created. I like that you made this idempotent! But probably not necessary.